tests/test_json.py

```diff
@@ -103,42 +103,51 @@ def test_simple_line_coverage(self):
         self._assert_expected_json_report(cov, expected_result)
 
     def test_context(self):
-        cov = coverage.Coverage(context="cool_test")
-        cov.config.json_show_contexts = True
-        expected_result = {
-            'meta': {
-                "version": coverage.__version__,
-                "branch_coverage": False,
-                "show_contexts": True,
-            },
-            'files': {
-                'a.py': {
-                    'executed_lines': [1, 2],
-                    'missing_lines': [3],
-                    'excluded_lines': [],
-                    "contexts": {
-                        "1": [
-                            "cool_test"
-                        ],
-                        "2": [
-                            "cool_test"
-                        ]
-                    },
-                    'summary': {
-                        'excluded_lines': 0,
-                        'missing_lines': 1,
-                        'covered_lines': 2,
-                        'num_statements': 3,
-                        'percent_covered': 66.66666666666667
+        for relative_files in [False, True]:
```

Comment:
easier to look at this diff at https://github.com/nedbat/coveragepy/pull/902/files?w=1 (hiding whitespace changes)

i tried to use pytest parametrize but it doesn't work with unitttest tests

Reply:
In cases like this, i've used helper functions.  I'll make that change.  I really appreciate that you updated the tests, and thought about how best to update them. Thanks!

Reply:
Thank you!